guide/src/pypy.md

```diff
@@ -0,0 +1,11 @@
+# PyPy Support
+
+Using PyPy is supported via cpyext.
+
+Support is only provided for building rust extension for code running under PyPy. This means PyPy **cannot** be called from rust via cpyext. Note that there some differences in the ffi module between pypy and cpython. 
+ 
+This is a limitation of cpyext and supported for embedding cpyext is not planned.
+
+Compilation against PyPy is done by exporting the `PYTHON_SYS_EXECUTABLE` to a pypy binary or by compiling in a PyPy virtualenv.
```

Comment:
When I try `cargo build` after setting `export PYTHON_SYS_EXECUTABLE=$(which pypy3.5)`, I am seeing failures:

>    Compiling pyo3cls v0.6.0 (.../pyo3/pyo3cls)
error: could not find native static library `python3.5`, perhaps an -L flag is missing?

Similar results for `pypy3.6`. Is this something I'm doing wrong?

Reply:
This looks like a regression in the build script. It is supposed to look for `libpypy3-c` instead of `libpython3.x`, I'll look into it later today, thanks for the catch!

Reply:
Can we add at least a run with `cargo build` to the run, to prevent regressions like this in the future? If we can't say that the tests will pass, we should at least be able to check that the project builds.

Reply:
added, good idea

Reply:
@pganssle could you please check that it's working on your machine?

Reply:
Still broken for me with your latest commits, plus now I'm seeing some warnings about unused imports and such:

> error: could not find native static library `pypy3-c`, perhaps an -L flag is missing?

If this is not some issue on my local environment, then it also seems like the tests are not effectively testing the `cargo build` command on PyPy. I'm using [pyenv](https://github.com/pyenv/pyenv) with `pypy3.5-7.0.0` and using `export PYTHON_SYS_EXECTUABLE=$(which pypy3.5)`. Perhaps the test suite needs to also export the `PYTHON_SYS_EXECUTABLE` variable?

Reply:
I've set cargo build  explicitly the env.
The examples tests are using `--cfg=PyPy` inside the setup.py.

Btw, what platform are you using? I cannot reproduce it locally.
It seems it's looking for the right dylib this time.
Where is it located in your environment? it should be named `libpypy3-c.dylib` or `libpypy3-c.so`.

Reply:
@pganssle it should be really fixed this time, and actually testing the build in the CI.
Thanks for lending a hand in testing this again, and sorry about it!